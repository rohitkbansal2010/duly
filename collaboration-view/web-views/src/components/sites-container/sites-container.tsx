export const SitesContainer: React.FC = ({ children }) => 
  (
    <>
      {children}
    </>
  );
