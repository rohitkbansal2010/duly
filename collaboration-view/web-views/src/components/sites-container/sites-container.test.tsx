import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';

import { SitesContainer } from './sites-container';

const mockDispatch = jest.fn();
jest.mock('react-redux', () =>
  ({
    ...jest.requireActual('react-redux'),
    useDispatch: () =>
      mockDispatch,
  }));


describe('SitesContainer', () => {
  let Component: ReturnType<typeof mount>;
  
  const TestComponent = () => 
    (
      <div data-testid="test-component">
        test
      </div>
    );
  
  beforeEach(() => {
    Component = mount(
      <SitesContainer>
        <TestComponent />
      </SitesContainer>
    );
  });

  it('should render correctly', () => {
    expect(toJson(Component)).toMatchSnapshot();
  });

  it('should render passed children', () => {
    expect(Component.contains(<TestComponent />)).toBe(true);
  });
});
