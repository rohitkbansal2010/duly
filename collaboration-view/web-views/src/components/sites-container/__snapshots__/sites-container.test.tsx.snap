// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SitesContainer should render correctly 1`] = `
<SitesContainer>
  <TestComponent>
    <div
      data-testid="test-component"
    >
      test
    </div>
  </TestComponent>
</SitesContainer>
`;
